protocols/v2/codec-sv2/src/encoder.rs

```diff
@@ -1,3 +1,47 @@
+//! # Stratum V2 Codec Encoder
```

Comment:
"Stratum V2 Codec Encoder" feels a bit redundant.

```suggestion
//! # Encoder
```